core/jazz_logs/test/test.js

```diff
@@ -6,6 +6,7 @@ const awsContext = require('aws-lambda-mock-context');
 const sinon = require('sinon');
 const logger = require("../components/logger.js");
 const request = require('request');
+const _ = require('lodash');
```

Comment:
We are reducing our dependence on third-party libraries. One of them is lodash. We are replacing all lodash usages in the code with js native functions. Please remove lodash and use native functions